README.md

```diff
@@ -1217,13 +1217,13 @@ While this guide explains the *what*, *why* and *how*, I find it helpful to see
       var directive = {
           restrict: 'EA',
           templateUrl: 'app/feature/example.directive.html',
-          scope: {
+		  scope: {},
```

Comment:
should this just be `scope: true`? I believe the source code shows this being a valid value